test/feature/StrongMode/Ellision.js

```diff
@@ -0,0 +1,15 @@
+// Options: --strong-mode
+
+'use strong'
+
+let a = [0, 1, 2, ];
+assert.equal(3, a.length);
+
+let [, b, , c] = [0, 1, 2, 3];
+assert.equal(1, b);
+assert.equal(3, c);
+
+// Not working ATM.
+// let d;
+// [, d] = [0, 1];
```

Comment:
Well, FP-style list iteration would be an extremely bad idea to apply to arrays, where you'd end up copying the tail array all the time...